flink-core/src/main/java/org/apache/flink/core/memory/MemorySegmentFactory.java

```diff
@@ -32,6 +36,7 @@
  */
 @Internal
 public final class MemorySegmentFactory {
```

Comment:
Maybe it is a Github issue, but this commit should occur before 167df1ef2da294b747cf7fd03cc8314599c6b5a5.

Reply:
it is a Github issue